### ISSUE TITLE
Auto-boot with power limits?

### ISSUE THREAD

**Issue by WonkerChonker** [PrimeO7/How-to-undervolt-AMD-RYZEN-5800X3D-Guide-with-PBO2-Tuner#33]
Hi,

Does there happen to be a way to boot with PPT, TDC, and EDC limits via the task scheduler method? My apologies if I missed it. 

Thanks!

**Comment by wdond086** [comment]
Yes. I read through the docs as well and actually tried the command but did not find a way to provide the PPT, TDC and EDC to the powershell command. Adding would be great.

**Comment by schmitt00** [comment]
I would also much like this to be implemented.
I have mine set to -100Hz which additionally lower temps and voltage but have to manually reapply every boot.

**Comment by lucassilvas1** [comment]
Any update on this? I've seen people suggest adding three values after the core values for PPT, TDC, and EDC, but on my end it doesn't work at all if I have those three values, not even the offsets will apply, it only works if I pass it the amount of values corresponding to how many cores I have.

Edit: Nevermind, if you want to set the limits, you need four values after the core offsets: PPT, TDC, EDC and FMAX (set it to 0 not to override it)
